woocommerce/woocommerce-admin
# Update DevDocs to be more Calypso-like

Fixes #2844.

This PR does the following:
- Separates Markdown and JSX in example code and documentation (allowing real JS imports and documentation when browsing components on GitHub)
- Style the DevDoc components closer to Calypso (including in the generation scripts)
- Symlink the `docs/components/packages` folder to `packages/components/src` to maintain Docsify usage
- Finish documenting the `AdvancedFilters` component

### Screenshots

**Before:**

![Screen Shot 2019-09-03 at 12 46 00 PM](https://user-images.githubusercontent.com/63922/64203646-d2233280-ce48-11e9-9ac0-98fc468fca6d.png)

**After:**

![Screen Shot 2019-09-03 at 12 43 53 PM](https://user-images.githubusercontent.com/63922/64203661-d9e2d700-ce48-11e9-9f69-2af55c67f1f6.png)

### Detailed test instructions:

- Go to WooCommerce > DevDocs
- Verify that all components render, examples function, doc formatting is good
- `npx docsify serve docs`
- Go to `localhost:3000`
- Verify the docs render and can be navigated



### Changelog Note:



Dev: refresh component documentation
